src/dc_msg.rs

```diff
@@ -472,7 +475,13 @@ pub fn dc_msg_load_from_db<'a>(msg: *mut dc_msg_t<'a>, context: &'a Context, id:
                 (*msg).type_0 = row.get(12)?;
                 (*msg).state = row.get(13)?;
                 (*msg).is_dc_message = row.get(14)?;
-                (*msg).text = row.get::<_, Option<String>>(15)?;
+                (*msg).text = Some(match row.get(15) {
```

Comment:
why not load it as bytes and try the conversion your self, that way you could return at least a utf8 lossy version

Reply:
I tried to but the ValueRef enum has no method to give me it raw. I can only get it as a &str. But maybe i'm overseeing something.

Reply:
EDIT: Got it. Thanks to @gwenn